mypy_self_check.ini

```diff
@@ -19,3 +19,4 @@ pretty = True
 always_false = MYPYC
 plugins = misc/proper_plugin.py
 python_version = 3.5
+exclude = /(mypy/typeshed|__pycache__|site-packages)/|/(\.[^/]+)/?$
```

Comment:
Nice!

If we always enable the defaults, this can be simplified to `/mypy/typeshed/`.